### ISSUE TITLE
[Feature]: SAM支持负面prompt吗

### ISSUE THREAD

**Issue by WhiteGL** [continue-revolution/sd-webui-segment-anything#187]
### Expected behavior

使用SAM和groundingDINO推理的时候后台是显示根据正向prompt和负向prompt得到结果的，但是插件界面没看到有输入负面prompt的窗口，是我没发现吗

**Comment by hacbachvotu** [comment]
yes, i expect the negative prompt because i only want to choose the face's skin, not all the face.
